hooks/pre/10-reset_feature.rb

```diff
@@ -81,9 +81,8 @@ def empty_mongo
   else
     Kafo::Helpers.execute(
       [
-        'systemctl stop rh-mongodb34-mongod',
-        "rm -f /var/lib/mongodb/#{mongo_config[:database]}*",
-        'systemctl start rh-mongodb34-mongod'
+        'systemctl start rh-mongodb34-mongod',
+        "mongo #{mongo_config[:database]} --eval 'db.dropDatabase();'"
```

Comment:
In the other command the order is reversed. I haven't done a lot with mongo and shell conventions suggest it shouldn't be a difference, but just to verify: there isn't, right?

Reply:
@ekohl the other command was not dropping the database correctly, for local the syntax is
```
mongo <db_name> --eval 'db.dropDatabase();'
```

Reply:
https://stackoverflow.com/questions/8857276/how-do-i-drop-a-mongodb-database-from-the-command-line

Reply:
I was ambiguous. With other I meant the remote. There it's:
```
   cmd = "mongo #{username} #{password} #{host} #{ssl} #{ca_cert} #{client_cert} --eval \"db.dropDatabase();\" #{config[:database]}" 
```

So `mongo --eval 'db.dropDatabase();' db` vs `mongo db --eval 'db.dropDatabase();'`

Again, this shouldn't matter but just making sure :)

Reply:
@ekohl It works both ways so set it to how I changed the first command to keep things similar and easier to follow, will post test results with the updated command in a separate comment.